manifest.yml

```diff
@@ -3,3 +3,7 @@ applications:
 - name: fec-web
   host: fec
   buildpack: python_buildpack
+env:
+  FEC_WEB_DEBUG: True
+  FEC_WEB_API_URL: 'http://fec-api-dev.cf.18f.us/'
```

Comment:
It makes sense to me for the deploy script to set the `DEBUG` and apis based on the second parameter (assuming you have enough information from "fec-web-dev" to convert to "fec-api-dev" etc.).

I see two ways to do this. 
1. Leave the manifest without these env vars, `cf push` then `cf set-env` and restart the app
2. Instead of `cf push`ing with this manifest, do something like: 
```bash
cp manifest.yml /tmp/
echo "env:" >> /tmp/manifest.yml
echo "    FEC_WEB_DEBUG: Some Determined Value" >>  /tmp/manifest.yml
echo "    FEC_WEB_API_URL: Some Other Value" >>  /tmp/manifest.yml
cf push "$2" -f /tmp/manifest.yml
```

There might also be a way to set env vars at `push` time, but I didn't see one.